client/state/action-types.js

```diff
@@ -219,6 +219,7 @@ export const HAPPINESS_ENGINEERS_FETCH_SUCCESS = 'HAPPINESS_ENGINEERS_FETCH_SUCC
 export const HAPPINESS_ENGINEERS_RECEIVE = 'HAPPINESS_ENGINEERS_RECEIVE';
 export const HAPPYCHAT_BLUR = 'HAPPYCHAT_BLUR';
 export const HAPPYCHAT_FOCUS = 'HAPPYCHAT_FOCUS';
+export const HAPPYCHAT_ELIGIBILITY_SET = 'HAPPYCHAT_ELIGIBILITY_SET';
```

Comment:
For context: there is the [happychat-client](https://github.com/Automattic/happychat-client/) project which is going to be the source of truth for Happychat customer facing code. In a few weeks, Calypso will be modified to use this library and the only code remaining will be Calypso-specific.

In that context, I've been namespacing the action names. At the moment, Calypso code doesn't have any Happychat user actions, but it will, and I've been using `HAPPYCHAT_USER_*` for thes. With that in mind, can this be renamed to `HAPPYCHAT_USER_ELIGIBILITY`?